src/app/cli/src/commands/sql_shell_command.rs

```diff
@@ -124,17 +128,30 @@ impl SqlShellCommand {
 
         Ok(())
     }
+
+    async fn run_datafusion_cli_command(&self) -> Result<(), CLIError> {
+        let mut print_options = PrintOptions {
+            format: PrintFormat::Table,
+            quiet: true,
+            maxrows: MaxRows::Unlimited,
+        };
+
+        let mut ctx = self.query_svc.create_session().await.unwrap();
+
+        exec::exec_from_repl(&mut ctx, &mut print_options)
+            .await
+            .map_err(|e| DataFusionError::External(Box::new(e)));
+
+        Ok(())
+    }
 }
 
 #[async_trait::async_trait(?Send)]
 impl Command for SqlShellCommand {
     async fn run(&mut self) -> Result<(), CLIError> {
         match (self.engine.as_deref(), &self.command, &self.url) {
+            (Some("datafusion"), None, None) => self.run_datafusion_cli_command().await,
             (Some("datafusion"), Some(_), None) => self.run_datafusion_command().await,
-            (Some("datafusion"), _, _) => Err(CLIError::usage_error(
-                "DataFusion engine currently doesn't have a shell and only supports single \
-                 command execution",
-            )),
             (Some("spark") | None, _, _) => self.run_spark_shell().await,
```

Comment:
Your PR comment says:
> made datafusion the default engine for kamu sql command

But when I run `kamu sql` or `kamu sql -c 'select 1;` I end up in the Spark SQL shell.